frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -312,7 +312,16 @@ export const StyledSidebarOpenContainer =
       alignItems: "center",
 
       [`@media print`]: {
-        position: "static",
+        position: "absolute",
+        left: 0,
+        top: 0,
+        marginTop: 0,
+
+        "& img": {
```

Comment:
Should we be more specific to target the img of the logo only?

Reply:
Good point, I have updated the selector to use the `stLogo` test id. Does this sound good?

Reply:
test-id wouldn't be good here. Either we add a class or we could directly link to `StyledLogo` here (preferred), e.g.:

https://github.com/streamlit/streamlit/blob/7aa818bfadbbbfabe2ea72f0ce763deba234b830/frontend/lib/src/components/core/Block/styled-components.ts#L143

Reply:
Great hint, I have updated it to ``[`& > ${StyledLogo}`]``